traffic_ops/traffic_ops_golang/deliveryservice/servers/servers.go

```diff
@@ -957,9 +957,12 @@ func (dss *TODSSDeliveryService) Read(h http.Header, useIMS bool) ([]interface{}
 		if version.Major > 3 && version.Minor >= 0 {
 			ds = ds.RemoveLD1AndLD2()
 			returnable = append(returnable, ds)
-		} else {
+		} else if version.Major > 2 {
 			dsV3 := ds.DowngradeToV3()
 			returnable = append(returnable, dsV3)
```

Comment:
this will return APIv3.1 structures in calls to v3.0 - gotta check the minor version

Reply:
yeah missed your previous comment, fixing this now